tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/SqlInjection/EFCoreTests.cs

```diff
@@ -8,6 +8,7 @@ namespace Samples.InstrumentedTests.Iast.Vulnerabilities.SqlInjection;
 
 // We cannot use localDB on linux and these calls cannot be mocked
 [Trait("Category", "LinuxUnsupported")]
+[Collection("Non-Parallel Collection")]
```

Comment:
Need to fix these too, sorry!
```suggestion
[Collection("IastEfCoreSqliteTests")]
```